src/client/src/orangehrmBuzzPlugin/components/PostActionCommentButton.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <oxd-icon-button class="" name="chat-text-fill" :with-container="true" />
+</template>
+<script>
+export default {
+  name: 'CommentButton',
+
+  props: {
+    postId: {
+      type: Number,
+      required: true,
+    },
+    buzzUserId: {
+      type: Number,
+      required: true,
+    },
```

Comment:
do we need this? I feel better to remove now. if you feel you need them later, add them at that point. Otherwise if design changed middle of the development, these codes may remains as ununsed

Reply:
Removd